test/print_pumipic_partition.cc

```diff
@@ -0,0 +1,85 @@
+#include <gmi_mesh.h>
+#include <apf.h>
+#include <apfMesh2.h>
+#include <apfMDS.h>
+#include <PCU.h>
+#include <lionPrint.h>
+#include <parma.h>
+
+#ifdef HAVE_SIMMETRIX
+#include <gmi_sim.h>
+#include <SimUtil.h>
+#include <MeshSim.h>
+#include <SimModel.h>
+#endif
+#include <apfZoltan.h>
+#include <pcu_util.h>
+#include <cstdlib>
+#include <fstream>
+
+int main(int argc, char** argv)
+{
+  MPI_Init(&argc,&argv);
+  PCU_Comm_Init();
+  lion_set_verbosity(1);
+  if ( argc != 5 && argc != 6) {
+    if ( !PCU_Comm_Self() )
+      printf("Usage: %s <model> <mesh> <number of ranks> <partition file prefix>\n", argv[0]);
```

Comment:
Changing 'number of ranks' to 'number of output parts' may be easier to understand.